buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/RebaselinePerformanceTests.groovy

```diff
@@ -40,7 +40,7 @@ class RebaselinePerformanceTests extends SourceTask {
         this.baseline = baseline
     }
 
-    static String rebaseline(String fileContent, String baseline) {
+    private static String rebaselineContent(String fileContent, String baseline) {
```

Comment:
This should not be private when it is referenced from the test.

Reply:
If you want to signal that it's not `private` because of testing then `@VisibleForTesting` might be a good option.

Reply:
Done